modules/Bio/EnsEMBL/Compara/PipeConfig/EG/WBParaSiteProteinTrees_conf.pm

```diff
@@ -1,7 +1,7 @@
 =head1 LICENSE
 
-See the NOTICE file distributed with this work for additional information
-regarding copyright ownership.
+Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+Copyright [2016-2020] EMBL-European Bioinformatics Institute
```

Comment:
Our header we have follows the latest Ensembl Software License policy (and avoids us to have to change it every year), so please revert this change.

Reply:
Done, thanks :)